src/modules/roc_core/target_posix/roc_core/time.cpp

```diff
@@ -17,6 +19,7 @@
 namespace roc {
 namespace core {
 
+#if defined(CLOCK_MONOTONIC) && defined(TIMER_ABSTIME)
```

Comment:
I think we need separate #ifdefs for CLOCK_MONOTONIC and TIMER_ABSTIME. E.g. it's possible that CLOCK_MONOTONIC is available, and TIMER_ABSTIME is not.